test/teslamate/vehicles/vehicle/charging_test.exs

```diff
@@ -76,6 +79,7 @@ defmodule TeslaMate.Vehicles.Vehicle.ChargingTest do
                       date: _,
                       charge_energy_added: 0.4,
                       rated_battery_range_km: 6.44,
+                      charge_rate: 10,
```

Comment:
these values should set to realisitc values

Reply:
As before

Reply:
as before